plugins/ngrok/provisioner.go

```diff
@@ -138,9 +146,9 @@ func getNgrokVersion() (string, error) {
 }
 
 func (f fileProvisioner) Deprovision(ctx context.Context, in sdk.DeprovisionInput, out *sdk.DeprovisionOutput) {
-	// nothing to do here: files get deleted automatically by 1Password CLI
+	// nothing to do here: files get deleted automatically by 1Password CLI and environment variables get wiped when process exits
 }
 
 func (f fileProvisioner) Description() string {
-	return "Config file aware provisioner. It will first check if an already existing config file is present."
+	return "If ngrok version is 3.2.1 or higher than provision ngrok credentials as environment variables NGROK_AUTH_TOKEN and NGROK_API_KEY otherwise config file aware provisioner. It will first check if an already existing config file is present."
```

Comment:
```suggestion
	return "If ngrok version is 3.2.1 or higher then provision ngrok credentials as the environment variables NGROK_AUTH_TOKEN and NGROK_API_KEY, otherwise use config file aware provisioner. It will first check if an already existing config file is present."
```